e2e/src/step_definitions/media-viewer.ts

```diff
@@ -452,3 +456,98 @@ Then('I expect custom toolbar button should be enabled', async () => {
 Then('I expect custom toolbar should be enabled', async () => {
   await page.waitForElement(by.className('customToolbar'));
 });
+
+When(/^The user clicks on the show comments panel toggle icon$/, async function () {
+  await genericMethods.sleep(2000);
+  await commentsPanelPage.clickShowCommentsToggleIcon();
+  await genericMethods.sleep(3000);
+
+  // await commentsPanelPage.clickOnSearchTab();
+
+});
+
+//I expect to see the comments filter and search tabs
+When(/^I expect to be able to click on the Filter Search And Comments Tab$/, async function () {
```

Comment:
when should be used with actions, like the user clicking on something. expectations should be done in then

Reply:
refactored